ansible_collections/arista/avd/docs/plugins/Modules_and_action_plugins/deploy_to_cv.md

```diff
@@ -0,0 +1,119 @@
+---
+# This title is used for search results
+title: arista.avd.deploy_to_cv
+---
+<!--
+  ~ Copyright (c) 2023-2024 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# deploy_to_cv
+
+!!! note
+    Always use the FQCN (Fully Qualified Collection Name) `arista.avd.deploy_to_cv` when using this plugin.
+
+!!! warning "This module is in **preview** mode"
+    This module is not guaranteed to have a backwards compatible interface.
+
+Deploy various objects to CloudVision
+
+## Synopsis
+
+The <code>arista.avd.deploy\_to\_cv</code> module is an Ansible Action Plugin providing the following capabilities\:
+
+\- Verify Devices are in the CloudVision inventory.
+\- Verify Devices are in the Inventory \& Topology Studio.
+\- Update the Device hostname in the Inventory \& Topology Studio as needed.
+\- Create Workspace and build, submit, abandon as needed.
+\- Deploy EOS configurations using \"Static Configlet Studio\".
+\- Create and associate Device and Interface Tags.
+\- Approve, run, cancel Change Controls as needed.
```

Comment:
I assume these were intended to render as lists. but they are not rendering as lists.